src/index.ts

```diff
@@ -26,42 +27,46 @@ const run = async (): Promise<void> => {
+        console.log(`${error}, Could not send data`);
       }
     }
 
-    console.log(`Repo ID: ${prData.repositoryId}`);
-    console.log(`Ref of branch being merged: ${prData.ref}`);
-    console.log(`Ref of branch being merged into: ${prData.baseRef}`);
-    console.log(`SHA of merge commit: ${prData.sha}`);
-    console.log(`PR creator: ${prData.actor}`);
-    console.log(`Time PR created: ${prData.timestamp}`);
-    console.log(`Lines percent: ${prData.coverage.lines.percent}`);
-    console.log(`Functions percent: ${prData.coverage.functions.percent}`);
-    console.log(`Branches percent: ${prData.coverage.branches.percent}`);
+    if (actionDebug === 'true') {
+      console.log(`Repo ID: ${prData.repositoryId}`);
+      console.log(`Ref of branch being merged: ${prData.ref}`);
+      console.log(`Ref of branch being merged into: ${prData.baseRef}`);
+      console.log(`SHA of merge commit: ${prData.sha}`);
+      console.log(`PR creator: ${prData.actor}`);
+      console.log(`Time PR created: ${prData.timestamp}`);
+      console.log(`Lines percent: ${prData.coverage.lines.percent}`);
+      console.log(`Functions percent: ${prData.coverage.functions.percent}`);
+      console.log(`Branches percent: ${prData.coverage.branches.percent}`);
 
-    if (prData.coverage.lines.diff || prData.coverage.lines.diff === 0) {
-      console.log(`Lines difference: ${prData.coverage.lines.diff}`);
-    }
+      if (prData.coverage.lines.diff || prData.coverage.lines.diff === 0) {
+        console.log(`Lines difference: ${prData.coverage.lines.diff}`);
+      }
 
-    if (prData.coverage.functions.diff || prData.coverage.functions.diff === 0) {
-      console.log(`Functions difference: ${prData.coverage.functions.diff}`);
-    }
+      if (prData.coverage.functions.diff || prData.coverage.functions.diff === 0) {
+        console.log(`Functions difference: ${prData.coverage.functions.diff}`);
+      }
 
-    if (prData.coverage.branches.diff || prData.coverage.branches.diff === 0) {
-      console.log(`Branches Difference: ${prData.coverage.branches.diff}`);
-    }
+      if (prData.coverage.branches.diff || prData.coverage.branches.diff === 0) {
+        console.log(`Branches Difference: ${prData.coverage.branches.diff}`);
+      }
 
-    if (prData.message) {
-      console.log(`Message: ${prData.message}`);
-    }
+      if (prData.message) {
+        console.log(`Message: ${prData.message}`);
+      }
 
-    if (prData.pullRequest) {
-      console.log(`Pull Request Number: ${prData.pullRequest}`);
-    }
+      if (prData.pullRequest) {
+        console.log(`Pull Request Number: ${prData.pullRequest}`);
+      }
 
-    if (context.payload.pull_request) {
-      makePullRequestComment(prData.message, prData.coverage);
+      if (context.payload.pull_request) {
+        makePullRequestComment(prData.message, prData.coverage);
+      }
+    } else if (actionDebug === 'false' || !actionDebug) {
+      console.log(prData.message);
```

Comment:
I think you can probably just put this under  the if actionDebug===true statement and remove the elseif